src/main/java/uk/gov/register/presentation/view/RegisterDetailView.java

```diff
@@ -49,4 +50,9 @@ public RegisterDetail getRegisterDetail() {
     public CsvRepresentation<RegisterDetail> csvRepresentation() {
         throw new NotImplementedException("Not implemented");
     }
+
+    @Override
+    public Model turtleRepresentation() {
+        throw new NotImplementedException("Not implented");
```

Comment:
why are we implementing RepresentaionView if we don't want to actually implement it?